### DISCUSSION TITLE
启用 `v1 风格设置面板`时打开`自定义顶栏`内的`布局设置`面板后关闭脚本总的侧边设置面板

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#2642] [功能建议 / Ideas]
不然自定义顶栏有一部分是看不见的

**Comment by the1812** [comment]
是因为开着 v1 风格设置面板吗

**Reply by Geeyun-JY3** [reply]
yep，毕竟 v1 就是收起的。我改改标题吧

**Comment by Geeyun-JY3** [comment]
今天偶然发现启用 `v1 风格设置面板`时打开`自定义顶栏`内的`布局设置`面板后脚本总的侧边设置面板不会关闭，所以这算是又变回老样子了？

**Reply by the1812** [reply]
想不起来当时是怎么处理的了, 点一下顶栏的设置弹窗倒是能把设置面板关掉

**Reply by Geeyun-JY3** [reply]
相关 commit：<https://github.com/the1812/Bilibili-Evolved/commit/ecb4d190d139578e025c4c356fcdf7ec5070ce3e>？

所以还会再修一次吗？我甚至觉得可以进一步优化：启用 `v1 风格设置面板`时，打开`自定义顶栏`内的`布局设置`面板后关闭脚本总的侧边设置面板，关闭`自定义顶栏`内的`布局设置`面板后自动打开脚本总的侧边设置面板。

（其实当时我没注意到可以通过光标悬停在设置面板的透视图标来看到`自定义顶栏`的全貌，在`自定义顶栏`内的`布局设置`面板或`自定义顶栏`设置项简介里加一行提示好像也可以是一种解决方案。）

**Reply by Geeyun-JY3** [reply]
不过看更新日志的说法，现在这效果也算得上是符合描述？
https://github.com/the1812/Bilibili-Evolved/blob/7800adbb02ef884e269ce82c9c8564a17662c279/CHANGELOG.md?plain=1#L1229
设置面板确实会在**操作**`自定义顶栏`的设置（准确地说是`自定义顶栏`内的`布局设置`面板）时自动关闭。
那没事了（

至于我上面提到的优化，我最近有了进一步的想法。我还是整理一下后另开一个讨论吧。
